client/keys/export.go

```diff
@@ -0,0 +1,45 @@
+package keys
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/input"
+)
+
+func exportKeyCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export <name>",
+		Short: "Export private keys",
+		Long:  `Export a private key from the local keybase in ASCII-armored encrypted format.`,
+		Args:  cobra.ExactArgs(1),
+		RunE:  runExportCmd,
+	}
+	return cmd
+}
+
+func runExportCmd(_ *cobra.Command, args []string) error {
+	kb, err := NewKeyBaseFromHomeFlag()
+	if err != nil {
+		return err
+	}
+
+	buf := input.BufferStdin()
+	decryptPassword, err := input.GetPassword("Enter passphrase to decrypt your key:", buf)
+	if err != nil {
+		return err
+	}
+	encryptPassword, err := input.GetPassword("Enter passphrase to encrypt the exported key:", buf)
```

Comment:
As we discussed earlier, maybe we should open another issue for this, however, I think it would be good to improve `input.GetPassword` to ensure that something like `12345678` is not accepted.

Reply:
@zmanian ping